README.md

```diff
@@ -250,7 +250,8 @@ python -m venv venv && . venv/bin/activate
 python -m pip install --upgrade pip
 
 # install requirements
-pip install -r requirements.txt -r requirements-test.txt
```

Comment:
Please keep this even if empty, it doesn't hurt and if in a future there will be some new requirement it will already work in the CI.